### DISCUSSION TITLE
Pushing to foreign repository

### DISCUSSION THREAD

**Discussion by matthew-github-123** [community/community#23898] [New to GitHub]
<p>I am trying to push changes to a foreign repository via a push request (this is what is mentioned in the readme documentation). However I get:</p>
<p>ERROR: Permission to (foreign repository) denied to (my github username).<br>
fatal: Could not read from remote repository.</p>
<p>Please make sure you have the correct access rights and the repository exists.</p>
<p>I do have a SSH public key in my own github account.</p>
<p>Cheers</p>

**Comment by kingthorin** [comment, marked as answer]
<p>You push to your fork of the original repo, then open a pull request (via the web ui).</p>

**Comment by matthew-github-123** [comment]
<p>Thank you, this was great!</p>
